extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -0,0 +1,1691 @@
+          "properties": {
+            "height": {
+              "bufferView": 0
+            },
+            "birdCount": {
+              "bufferView": 1
+            },
+            "species": {
+              "bufferView": 2,
+              "stringOffsetBufferView": 3
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+`class` is the ID of the class in the schema. `count` is the number of features in the feature table, as well as the length of each property array. Property arrays are stored in glTF buffer views and use the binary encoding defined in the [Table Format](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata#table-format) section of the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata).
+
+Each buffer view `byteOffset` must be aligned to a multiple of 8 bytes. If the buffer view's buffer is the GLB-stored `BIN` chunk the byte offset is measured relative to the beginning of the GLB. Otherwise it is measured relative to the beginning of the buffer.
+
+### Feature Textures
+
+Feature textures (not to be confused with [Feature ID Textures](#feature-id-texture)) use textures rather than parallel arrays to store values. Feature textures are accessed directly by texture coordinates, rather than feature IDs. Feature textures are especially useful when texture mapping high frequency data to less detailed 3D surfaces.
+
+Feature textures use the [Raster Format](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata#raster-format) of the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata) with a few additional constraints:
+
+* A scalar property cannot be encoded into multiple channels. For example, it is not possible to encode a `UINT32` property in an `RGBA8` texture.
+* Components of fixed-length array properties must be separate channels within the same texture.
+* Variable-length arrays are not supported.
+
+Additionally, the data type and bit depth of the image must be compatible with the property type. An 8-bit per pixel RGB image is only compatible with `UINT8` or normalized `UINT8` properties, and array properties thereof with three components or less. Likewise, a floating point property requires a floating point-compatible image format like KTX2 which may require additional extensions.
+
+Feature textures are defined with the following steps:
+
+1. A class is defined in the root `EXT_feature_metadata` extension object. This is used to describe the metadata in the texture.
+2. A feature texture is defined in the root `EXT_feature_metadata.featureTextures` object. This must reference the class ID defined in step 1.
+3. A feature texture is associated with a primitive by listing the feature texture ID in the `primitive.EXT_feature_metadata.featureTextures` array.
+<img src="figures/feature-texture.png"  alt="Feature Texture" width="500">
```

Comment:
meeting notes: let's open PR and review implications

Reply:
The main implication of this change is the heavy use of upward references, but there's precedent with the texCoord property in textureInfo. But overall I think this is a very helpful restructuring.

Reply:
In https://github.com/CesiumGS/glTF/pull/14 we've gone partway here, with primitives pointing to Feature Textures by index rather than name. 

In https://gist.github.com/donmccurdy/dbd6b5b5baa217571c31f8ba9bf798b8 I've drafted what I think it might look like to go the rest of the way, specifying Feature Tables by index. I didn't go as far as moving the Feature ID storage into a root-level array, but kept that inlined (in a somewhat more compact form) within the primitive or node extension. If this looks like a good direction I will open a PR for those changes.

Reply:
I left some notes in the gist but I like the proposal. Good to open a PR!

Reply:
Resolved in https://github.com/CesiumGS/glTF/pull/14, thanks!